index.html

```diff
@@ -1,143 +1,252 @@
-      </div>
-      <div class="cardtwo">
-        <h5 class="topcenter">Profesional Art Printing Data</h5>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="language">
-          <li><button class="button button6">html</button></li>
-          <li><button class="button button7">bootstrap</button></li>
-          <li><button class="button button8">Ruby</button></li>
+        <ul class="listthree">
+          <li class="heading"><i class="fa-solid fa-circle"></i>Skills</li>
+          <li class="sub-list">Codekit</li>
+          <li class="sub-list">Github</li>
+          <li class="sub-list">Codepen</li>
+          <li class="sub-list">Gitlab</li>
+          <li class="sub-list">Terminal</li>
         </ul>
-        <button class="button button9">See project</button>
       </div>
-      <div class="cardtwo">
-        <h5 class="topcenter">Profesional Art Printing Data</h5>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="language">
-          <li><button class="button button6">html</button></li>
-          <li><button class="button button7">bootstrap</button></li>
-          <li><button class="button button8">Ruby</button></li>
-        </ul>
-        <button class="button button9">See project</button>
+    </section>
+    <footer class="contact-form">
+      <div class="primary-text">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
       </div>
-      <div class="cardtwo">
-        <h5 class="topcenter">Profesional Art Printing Data</h5>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <ul class="language">
-          <li><button class="button button6">html</button></li>
-          <li><button class="button button7">bootstrap</button></li>
-          <li><button class="button button8">Ruby</button></li>
-        </ul>
-        <button class="button button9">See project</button>
+      <div class="contact-form-elements">
+        <form action="https://formspree.io/f/xknypbwk" method="post">
+          <input
+            type="text"
+            name="full-name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <br />
+          <input
+            type="email"
+            name="email-address"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="text-area"
+            id="text-area"
+            cols="30"
+            rows="10"
+            placeholder="Enter text here"
+            maxlength="500"
+          ></textarea>
+          <input type="button" value="Get in touch"/>
```

Comment:
- [ ] [OPTIONAL] You can change the cursor to a pointer on hover over the button, it will make your design more detail-oriented to the provided one